internal/controller/evictionautoscaler_controller.go

```diff
@@ -137,7 +137,14 @@ func (r *EvictionAutoScalerReconciler) Reconcile(ctx context.Context, req ctrl.R
 		metrics.BlockedEvictionCounter.WithLabelValues(EvictionAutoScaler.Namespace, pdb.Name).Inc()
 
 		// Track scaling opportunity
-		metrics.ScalingOpportunityCounter.WithLabelValues(EvictionAutoScaler.Namespace, EvictionAutoScaler.Spec.TargetName, metrics.ScaleUpAction).Inc()
+		// Determine signal for scaling opportunity
+		signalLabel := metrics.PDBBlockedSignal
+		if pdb.Spec.MinAvailable != nil && int32(pdb.Spec.MinAvailable.IntValue()) == pdb.Status.DesiredHealthy {
```

Comment:
hyper nit. cast up to int64 usually safer. Not like we're ever going to get a billion replicas though.